src/IMenuGraphr.ts

```diff
@@ -263,6 +263,11 @@ export interface IMenuSchema extends IMenuBase {
      * How the menu should be positioned within its container.
      */
     position?: IMenuSchemaPosition;
+
+    /**
+     * Whether or not the last selected index should be saved.
+     */
+    saveIndex?: boolean;
```

Comment:
... turns out the error remains whether or not the property is in `IListMenuSchema`. I thought it went away. It only shows up in Visual Studio Code, no problem when building.